website/content/docs/connect/cluster-peering/create-manage-peering.mdx

```diff
@@ -7,24 +7,28 @@ description: >-
 
 # Create and Manage Peering Connections
 
-~> **Cluster peering is currently in technical preview:** Functionality associated with cluster peering is subject to change. You should never use the technical preview release in secure environments or production scenarios. Features in technical preview may have performance issues, scaling issues, and limited support.
+~> **Cluster peering is currently in beta:** Functionality associated with cluster peering is subject to change. You should never use the beta release in secure environments or production scenarios. Features in beta may have performance issues, scaling issues, and limited support.
 
-A peering token enables cluster peering between different datacenters. Once you generate a peering token, you can use it to establish a connection between clusters. Then you can export services and authorize other clusters to call those services.
+A peering token enables cluster peering between different datacenters. Once you generate a peering token, you can use it to establish a connection between clusters. Then you can export services and create intentions so that peered clusters can call those services.
+
+## Create a peering connection
```

Comment:
There are some new security features implemented last week that need to be accounted for here and in the API docs for peering:
* Every time a peering token is generated there is also a one-time-use establishment secret that is created and embedded in the token. Re-generating a peering token will invalidate the last establishment secret. Meaning that to establish a peering the latest token must be used.
* `/peering/establish` can only be dialed once for a given peering token. After establishing the peering a peering token cannot be re-used and a new one must be generated to re-establish the peering (if needed).